lib/livebook/runtime/definitions.ex

```diff
@@ -411,6 +416,29 @@ defmodule Livebook.Runtime.Definitions do
       Exqlite.query!(conn, "PRAGMA table_list", [])\
       """,
       packages: [kino_db, exqlite]
+    },
+    %{
+      type: :file_action,
+      file_types: [".xlsx"],
```

Comment:
nice work!

can we add `".xlsm"` here too?

it's the same XML format as xlsx but possibly containing VBA macros, `XlsxReader` also parses these files.

thanks!

Reply:
Hopefully today after work i will have some time to add xlsm and the proposed changes.